x/tally/module.go

```diff
@@ -144,5 +146,13 @@ func (am AppModule) BeginBlock(_ context.Context) error {
 
 // EndBlock returns the end block logic for the tally module.
 func (am AppModule) EndBlock(ctx context.Context) error {
+	start := time.Now()
+	defer func() { telemetry.SetGauge(float32(time.Since(start).Seconds()), "seda_tally_end_block_time") }()
```

Comment:
There is a time measure metric provided by SDK (ex: https://github.com/cosmos/cosmos-sdk/blob/738f288181180d2e77e04b38871c29cbcf97c5dc/x/gov/keeper/abci.go#L24)
I think this should be useful here

Reply:
I used that at first but it didn't play as nicely on the Prometheus side. I don't think it was showing up in Prometheus with that approach? But I don't remember at the moment.

Reply:
ah I see. We can take a look together tmrw too. But I think it'd be nice to be able to get all the metrics including the SDK ones in a uniform way

Reply:
Ah ok I figured how to make it work better with prometheus using that approach. I can push that change now!